static/elements/chromedash-myfeatures-page.js

```diff
@@ -64,9 +81,9 @@ class ChromedashMyFeatures extends LitElement {
 
         <chromedash-feature-table
           query="${query}"
-          ?signedIn=${Boolean(this.signedInUser)}
-          ?canEdit=${this.canEdit}
-          ?canApprove=${this.canApprove}
+          ?signedIn=${Boolean(this.user)}
+          ?canEdit=${this.user && this.user.can_edit}
```

Comment:
I like the way these expressions came out.

Reply:
Yes, it turns out to be very readable!